src/Toml/Printer.hs

```diff
@@ -12,12 +14,43 @@ import qualified Data.HashMap.Strict as HashMap
 import qualified Data.List.NonEmpty as NonEmpty
 import qualified Data.Text as Text
 
+-- Tab is equal to 2 spaces now.
 tab :: Int -> Text
 tab n = Text.cons '\n' (Text.replicate (2*n) " ")
 
+{- | Converts 'TOML' type into 'Text'.
+
+For example, this
+
+@
+TOML
+    { tomlPairs  = HashMap.fromList [(Key "title", String "TOML example")]
+    , tomlTables = HashMap.fromList
+          [( TableId (NonEmpty.fromList ["example", "owner"])
+           , TOML
+                 { tomlPairs  = HashMap.fromList [(Key "name", String "Kowainik")]
+                 , tomlTables = mempty
+                 , tomlTableArrays = mempty
+                 }
+           )]
+    , tomlTableArrays = mempty
+    }
+@
+
+will be translated to this
+
+@
+title = "TOML Example"
+
+[example.owner]
+  name = "Kowainik"
+
+@
+-}
 toml2Text :: TOML -> Text
 toml2Text = toml2TextInd 0
 
+-- | Converts 'TOML' into 'Text' with the given indent.
 toml2TextInd :: Int -> TOML -> Text
```

Comment:
I guess we can export this function as well. In future we will have `PrintOptions` data type with `defaultOptions`.